src/modules/stakewise/shared/api/status.go

```diff
@@ -1,9 +1,29 @@
 package swapi
 
-import (
-	"github.com/nodeset-org/eth-utils/beacon"
+type BeaconStatus string
+type NodesetStatus string
+
+const (
+	WithdrawalDone     BeaconStatus = "WithdrawalDone"
+	WithdrawalPossible BeaconStatus = "WithdrawalPossible"
+	ExitedSlashed      BeaconStatus = "ExitedSlashed"
+	ExitedUnslashed    BeaconStatus = "ExitedUnslashed"
+	ActiveSlashed      BeaconStatus = "ActiveSlashed"
+	ActiveExited       BeaconStatus = "ActiveExited"
+	ActiveOngoing      BeaconStatus = "ActiveOngoing"
+	PendingQueued      BeaconStatus = "PendingQueued"
+	PendingInitialized BeaconStatus = "PendingInitialized"
+	NotAvailable       BeaconStatus = "NotAvailable"
+)
+
+const (
+	RegisteredToStakewise NodesetStatus = "RegisteredToStakewise"
+	UploadedStakewise     NodesetStatus = "UploadedStakewise"
+	UploadedToNodeset     NodesetStatus = "UploadedToNodeset"
+	Generated             NodesetStatus = "Generated"
 )
 
-type ActiveValidatorsData struct {
-	ActiveValidators []beacon.ValidatorPubkey `json:"pubkeys"`
+type ValidatorStatusData struct {
+	BeaconStatus  map[string]BeaconStatus  `json:"beaconStatus"`  // string => beacon.ValidatorPubkey
```

Comment:
Can you use the actual `ValidatorPubkey` object as the key here instead of a string? That way you don't have to keep parsing / unparsing and dealing with the 0x prefix.